dashboard/src/modules/components/OverviewComponent/SavedRunsComponent.jsx

```diff
@@ -151,10 +150,10 @@ const SavedRunsComponent = () => {
                       {rowActions ? (
                         <ActionsColumn
                           items={rowActions}
-                          isDisabled={
-                            item?.metadata[DATASET_OWNER] !==
-                            loginDetails?.username
-                          }
+                          // isDisabled={
```

Comment:
Can we just remove this code instead of leaving it commented out?

Reply:
Same follows here. Once, user details is updated in the API can be uncommented.

Reply:
See my previous comment.

Reply:
Removed it